src/app/features/issue/providers/gitlab/gitlab-common-interfaces.service.ts

```diff
@@ -69,9 +69,12 @@ export class GitlabCommonInterfacesService implements IssueServiceInterface {
     return this._getCfgOnce$(projectId).pipe(
       switchMap((gitlabCfg) =>
         this.isEnabled(gitlabCfg) && gitlabCfg.isSearchIssuesFromGitlab
-          ? this._gitlabApiService
-              .searchIssueInProject$(searchTerm, gitlabCfg)
-              .pipe(catchError(() => []))
+          ? this._gitlabApiService.searchIssueInProject$(searchTerm, gitlabCfg).pipe(
+              catchError((err) => {
+                console.log('Error searching issue', err);
```

Comment:
```suggestion
                console.error('Error searching issue', err);
```

Reply:
Would you think that providing feedback to the user (e.g. via a snack message) might be a good idea here?

Reply:
I don't, this issue will most likely accure when there is a problem parsing an issue because of api change or programming error - so lets not normalize it with a message but send an error so the person working on the issue could see the priblem